### PR TITLE
Fix UBL writer for debit payment

### PR DESCRIPTION
**Pull request by Tr1ckMan**
Removal of obsolete debitor account details from UBL export to comply with CR-440 and CR-446.
Test was added.

### CODE REVIEW

File: ZUGFeRD.Test/XRechnungUBLTests.cs [hunk 1]
```diff
@@ -602,7 +602,92 @@ public void TestPartyIdentificationShouldNotExist()
...
+
+            using (var stream = new MemoryStream())
+            {
+                d.Save(stream, ZUGFeRDVersion.Version23, Profile.XRechnung, ZUGFeRDFormats.UBL);
+                stream.Seek(0, SeekOrigin.Begin);
+                
+                // test the raw xml file
+                string content = Encoding.UTF8.GetString(stream.ToArray());
+
+                Assert.IsFalse(Regex.IsMatch(content, @"<cac:PaymentMandate.*>.*<cbc:Name.*><\/cac:PaymentMandate>", RegexOptions.Singleline));
```
[the remark this answers is not in the document]
Comment (Tr1ckMan) [reply ZUGFeRD.Test/XRechnungUBLTests.cs:688]: I hope I understand you correctly.
IMHO
If the test reads the XML with the reader(InvoiceDescriptor), 2 components are tested. If the test then fails, it is not clear whether the error is in the writer or in the reader. This can also result in a false positive. If the writer writes the value although it should not be and the reader does not read the field, the test is green. In this case, the code would even be wrong in the writer and reader.
Long story short. 😊
As a system test, this is the right way to go.
Here as a unit test, this unnecessarily obscures the error location.
